mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java

```diff
@@ -119,8 +119,8 @@ private static X509Certificate asCert() throws CertificateException {
     @CommandLine.Option(names = "--supplied-saml-encryption-cert-file", description = "Public X509 cert for SAML encryption certificate supplied manually")
     private File manuallySuppliedSamlEncryptionsCert;
 
-    @CommandLine.Option(names = "--validityDays", description = "How many days the metadata is valid for (default 30 days)")
-    private Integer validityDays = 30;
+    @CommandLine.Option(names = "--validityTimestamp", description = "Expiry timestamp for metadata using ISO8601, e.g (2015-05-24T19:30:26.624Z)")
```

Comment:
This is going to need a change in the Go controller, too. Assume Phil and Chris know about it 👍